docs/whatsnew/latest.rst

```diff
@@ -1,12 +1,12 @@
 :orphan:
 
-What's new in revision 0.6.3
+What's new in revision 0.6.4.dev
 ---------------------------------------------------------------------------------------
 
-These are the changes in SpectroChemPy-0.6.3.
+These are the changes in SpectroChemPy-0.6.4.dev.
 See :ref:`release` for a full changelog including other versions of SpectroChemPy.
 
 New features
 ~~~~~~~~~~~~
 
-* add `nnl` and `pnnls` solvers to `MCRALS`
+* add `FastICA` analysis module
```

Comment:
FYI you normally don't need to change "latest.rst" nor "index.rst" as they are anyway generated automatically from "changelog.rst" when you commit!

Reply:
yes, I think I didn't change it myself, but I will check aniother time :-)